lib/profile/view/profile_page.dart

```diff
@@ -16,163 +17,53 @@ class _ProfileScreenState extends State<ProfileScreen> {
   @override
   Widget build(BuildContext context) {
     final l10n = context.l10n;
+    final theme = Theme.of(context); 
     return Scaffold(
+      backgroundColor: theme.colorScheme.onBackground,
       body: LayoutBuilder(
         builder: (context, BoxConstraints box) {
           return Padding(
             padding: EdgeInsets.only(
-              top: box.maxHeight * 0.1,
-              left: box.maxWidth * 0.03,
-              right: box.maxWidth * 0.03,
+              top: box.maxHeight * 0.09,
+              left: box.maxHeight * 0.03,
+              right: box.maxHeight * 0.03,
             ),
             child: Column(
```

Comment:
- Wrap your column in a `SingleChildScrollView()` to resolve the bottom overflow
- The Profile title should be increased too, `size: 32px, weight: 600 color: #181818`
- Also the screen background color should be htis shade of white `#F7F7F7` as on the design

![pr-rv2](https://user-images.githubusercontent.com/17765231/222723282-e2488ba1-11d2-4afa-a881-c05fafbe5556.jpeg)